static/specification.html

```diff
@@ -93,34 +106,220 @@
+              }
+            }
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Monetizing media
+        </h3>
+        <p>
+          The following example shows how to monetize various types of media
+          using different [=payment pointers=].
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;video src="myvideo" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/video/pay"&gt;
+            &lt;/video&gt;
+
+            &lt;audio src="music.mp3" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/audio/pay"&gt;
+            &lt;/audio&gt;
+
+            &lt;picture srcset="cat.jpeg" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/images/pay"&gt;
+            &lt;/picture&gt;
+          </pre>
+        </aside>
+      </section>
+    </section>
+    <section id="model">
       <h2>
-        Terminology
+        Model
       </h2>
       <dl>
         <dt>
-          <dfn data-lt="provider">Web Monetization provider</dfn>:
+          <dfn data-lt="monetized">Monetization</dfn>:
         </dt>
         <dd>
-          The party making payments on behalf of the user. A Web Monetization
-          provider uses the [[[SPSP]]] (<abbr title=
-          "Simple Payment Setup Protocol">SPSP</abbr>) on top of the
-          [[[Interledger]]] (<abbr title=
-          "Interledger Protocol">ILP</abbr>)/[[[STREAM]]] to provide a
-          high-level way to send money and data.
+          Payments made by a user to a website, facilitated through a
+          [=monetization provider=].
         </dd>
         <dt>
-          <dfn>Web Monetization receiver</dfn>:
+          <dfn data-lt="provider">Monetization provider</dfn>:
         </dt>
         <dd>
-          The party receiving payments on behalf of the website.
+          The party making payments on behalf of the user. A monetization
+          provider leverages the [[[Interledger]]] suite of protocols and
+          technologies (e.g., [[Open Payments]] based on [[STREAM]]) to provide
+          a high-level way to pay a [=monetization receiver=].
         </dd>
         <dt>
-          <dfn>Payment end-point</dfn>:
+          <dfn>Monetization receiver</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a conforming HTTP <a data-cite="SPSP#specification">SPSP
-          end-point</a>.
+          The party receiving payments on behalf of the website, whose details
+          are provided by a [=payment pointer=].
+        </dd>
+        <dt>
+          <dfn>Payment Pointer</dfn>:
+        </dt>
+        <dd>
+          A [=URL=] to an <a data-cite="Open Payments#specification">Open
+          Payments API entry-point</a> (i.e., a JSON resource containing
```

Comment:
Is it really just an Open Payments Payment Pointer? Should we include a warning that SPSP is deprecated?

Reply:
I'm not sure it's needed. I wouldn't put it here specifically if we do add it.

Reply:
Maybe just "API entry-point", dropping the "Open Payments" part?

Reply:
@adrianhopebailie 👆

Reply:
Why omit Open Payments? 
If this is just an "API entry-point" then how is an implementation supposed to know what API is implemented?

Reply:
I guess I'm thinking about backwards compatibility with SPSP. What if the account provider only supports SPSP? Then a GET to the payment pointer is redirected to `https://payment.pointer/alice/.well-known/pay`and it doesn't respond with the Open Payments json blob or the acts as entry point to a grant request.

Reply:
Since there is no mentioning of SPSP in the spec anymore, I guess we can keep it as is.